com.unity.multiplayer.mlapi/Editor/NetworkedAnimatorEditor.cs

```diff
@@ -1,12 +1,12 @@
-﻿using System;
-using MLAPI.Prototyping;
-using UnityEditor.Animations;
-using UnityEngine;
+//using System;
+//using MLAPI.Prototyping;
+//using UnityEditor.Animations;
+//using UnityEngine;
 
 namespace UnityEditor
 {
-    [CustomEditor(typeof(NetworkedAnimator), true)]
-    [CanEditMultipleObjects]
+    //[CustomEditor(typeof(NetworkedAnimator), true)]
+    //[CanEditMultipleObjects]
```

Comment:
It is not clear how the tag removal fits with the singleton removal. And, if it does, should it stay in commented?

Reply:
This seems like an unrelated change that should be removed